firefox-ios/firefox-ios-tests/Tests/XCUITests/FakespotTests.swift

```diff
@@ -144,6 +144,80 @@ class FakespotTests: IphoneOnlyTestCase {
         validateExpandedSettingsSection()
     }
 
+    // https://testrail.stage.mozaws.net/index.php?/cases/view/2358892
+    // Smoketest
+    func testOptInNotificationLayout() {
+        if skipPlatform { return }
+        // Navigate to a product detail page on amazon.com page
+        reachReviewChecker()
+        validateOptInLayout("Amazon", "Walmart", "Best Buy")
+
+        // Navigate to a product detail page on walmart.com page
+        validateLayoutOnWalmartAndBestBuy("https://www.walmart.com", isWalmart: true, "Walmart", "Amazon", "Best Buy")
```

Comment:
I'm always a bit concern of using real sites for the smoketest.. let's monitor this closely not only for failures but for build times

Reply:
i am planning to do that, that was the only solution to cover those scenarios, if the test becomes unstable in the future i will remove it from the plan

Reply:
or mark it as only functional and maintain it to work